Source/Dafny/Reporting.cs

```diff
@@ -153,7 +153,7 @@ public abstract class ErrorReporter {
   }
 
   public abstract class BatchErrorReporter : ErrorReporter {
-    private readonly Dictionary<ErrorLevel, List<ErrorMessage>> allMessages;
+    protected readonly Dictionary<ErrorLevel, List<ErrorMessage>> allMessages;
```

Comment:
This might be something for the style-guide (i.e., `.editorconfig` settings). This `protected readonly` field starts with a lower-case letter, whereas `Rewriter.Reporter` starts with an upper-case one.

Reply:
It was already a warning, but fixed, thanks !